src/spikeinterface/sorters/basesorter.py

```diff
@@ -290,7 +288,8 @@ def get_result_from_folder(cls, output_folder):
 
         if bool(log["error"]):
             raise SpikeSortingError(
-                "Spike sorting failed. You can inspect the runtime trace in spikeinterface_log.json"
+                f"Spike sorting failed. You can inspect the runtime trace in {output_folder}/spikeinterface_log.json. "
+                f"Error trace: {log['error_trace']}"
```

Comment:
Here also I would put a print(log["error_trace"]) before an dkeep the error tarce small.